include/z64animation.h

```diff
@@ -106,8 +106,14 @@ typedef enum {
 // An option is to implement and use `ANIM_FLAG_OVERRIDE_MOVEMENT`.
 #define ANIM_FLAG_UPDATE_Y (1 << 1)
 
-// (player-only) Related to scaling an animation from/to child/adult
-#define ANIM_FLAG_PLAYER_2 (1 << 2)
+// When this flag is set, Player's root limb position adjustment as child is disabled.
+// Many of Player's animations are originally created for Adult Link. When playing those 
+// animations as Child Link without any adjustment, he will appear to be floating in the air.
+// However, if an animation is created specifically for Child Link, it is desireable to disable
+// this adjustment of the root position.
+// Note that this flag will be ignored if `ANIM_FLAG_UPDATE_XZ` or `ANIM_FLAG_UPDATE_Y` are also
+// set. The adjustment will be applied in this case regardless of this flag being enabled. 
```

Comment:
I feel like this explanation should somehow mention "scaling" since that seems to be the specific adjustment that is applied (right?). Something like "Many of Link's animations were originally created for Adult Link and the root limb position is scaled down to fit Child Link's smaller skeleton. When this flag is set, this scaling is disabled"

Reply:
Took it in a slightly different direction, lmk if its fine